rust/query-stats/README.md

```diff
@@ -0,0 +1,75 @@
+# Query stats
+
+## Prerequisites
+This example requires an installation of:
+
+- [x] Install the [IC SDK](../developer-docs/setup/install/index.mdx) at version 0.16.1 or higher.
+- [x] Download the following project files from GitHub: https://github.com/dfinity/examples/
+
+Begin by opening a terminal window.
+
+This is currently only for use from within DFINITY, as the replica shipped with `dfx` as well as nodes on 
+mainnet do not enable the query stats feature at the moment.
+
+Once enabled on mainnet, the following instructions will work without the `--network` argument.
+
+From within DFINITY, it can be ran as following (e.g. with a deployment to the `small_with_query_stats` Farm testnet)
+
+Variable `$BN_HOSTNAME` points at the hostname of a boundary node. For example, from field `['bn_aaaa_records']['url']` of the output of a Farm deployment
+
+## Installing the canister
+
+```
+dfx canister create --network="$BN_HOSTNAME" query_stats
```

Comment:
```suggestion
dfx canister create --network ic query_stats
```
Choose between mainnet or local and adapt this everywhere where `BN_HOSTNAME` shows up